test/fuzz/transformation_add_bit_instruction_synonym_test.cpp

```diff
@@ -288,174 +339,627 @@ TEST(TransformationAddBitInstructionSynonymTest, AddBitwiseSynonym) {
+        %235 = OpBitwiseXor %2 %233 %234
+
+        %236 = OpBitFieldUExtract %2 %7 %27 %6 ; extracts bit 22 from %7
+        %237 = OpBitFieldUExtract %2 %8 %27 %6 ; extracts bit 22 from %8
+        %238 = OpBitwiseXor %2 %236 %237
+
+        %239 = OpBitFieldUExtract %2 %7 %28 %6 ; extracts bit 23 from %7
+        %240 = OpBitFieldUExtract %2 %8 %28 %6 ; extracts bit 23 from %8
+        %241 = OpBitwiseXor %2 %239 %240
+
+        %242 = OpBitFieldUExtract %2 %7 %29 %6 ; extracts bit 24 from %7
+        %243 = OpBitFieldUExtract %2 %8 %29 %6 ; extracts bit 24 from %8
+        %244 = OpBitwiseXor %2 %242 %243
+
+        %245 = OpBitFieldUExtract %2 %7 %30 %6 ; extracts bit 25 from %7
+        %246 = OpBitFieldUExtract %2 %8 %30 %6 ; extracts bit 25 from %8
+        %247 = OpBitwiseXor %2 %245 %246
+
+        %248 = OpBitFieldUExtract %2 %7 %31 %6 ; extracts bit 26 from %7
+        %249 = OpBitFieldUExtract %2 %8 %31 %6 ; extracts bit 26 from %8
+        %250 = OpBitwiseXor %2 %248 %249
+
+        %251 = OpBitFieldUExtract %2 %7 %32 %6 ; extracts bit 27 from %7
+        %252 = OpBitFieldUExtract %2 %8 %32 %6 ; extracts bit 27 from %8
+        %253 = OpBitwiseXor %2 %251 %252
+
+        %254 = OpBitFieldUExtract %2 %7 %33 %6 ; extracts bit 28 from %7
+        %255 = OpBitFieldUExtract %2 %8 %33 %6 ; extracts bit 28 from %8
+        %256 = OpBitwiseXor %2 %254 %255
+
+        %257 = OpBitFieldUExtract %2 %7 %34 %6 ; extracts bit 29 from %7
+        %258 = OpBitFieldUExtract %2 %8 %34 %6 ; extracts bit 29 from %8
+        %259 = OpBitwiseXor %2 %257 %258
+
+        %260 = OpBitFieldUExtract %2 %7 %35 %6 ; extracts bit 30 from %7
+        %261 = OpBitFieldUExtract %2 %8 %35 %6 ; extracts bit 30 from %8
+        %262 = OpBitwiseXor %2 %260 %261
+
+        %263 = OpBitFieldUExtract %2 %7 %36 %6 ; extracts bit 31 from %7
+        %264 = OpBitFieldUExtract %2 %8 %36 %6 ; extracts bit 31 from %8
+        %265 = OpBitwiseXor %2 %263 %264
+
+        %266 = OpBitFieldInsert %2 %172 %175 %6 %6 ; inserts bit 1
+        %267 = OpBitFieldInsert %2 %266 %178 %7 %6 ; inserts bit 2
+        %268 = OpBitFieldInsert %2 %267 %181 %8 %6 ; inserts bit 3
+        %269 = OpBitFieldInsert %2 %268 %184 %9 %6 ; inserts bit 4
+        %270 = OpBitFieldInsert %2 %269 %187 %10 %6 ; inserts bit 5
+        %271 = OpBitFieldInsert %2 %270 %190 %11 %6 ; inserts bit 6
+        %272 = OpBitFieldInsert %2 %271 %193 %12 %6 ; inserts bit 7
+        %273 = OpBitFieldInsert %2 %272 %196 %13 %6 ; inserts bit 8
+        %274 = OpBitFieldInsert %2 %273 %199 %14 %6 ; inserts bit 9
+        %275 = OpBitFieldInsert %2 %274 %202 %15 %6 ; inserts bit 10
+        %276 = OpBitFieldInsert %2 %275 %205 %16 %6 ; inserts bit 11
+        %277 = OpBitFieldInsert %2 %276 %208 %17 %6 ; inserts bit 12
+        %278 = OpBitFieldInsert %2 %277 %211 %18 %6 ; inserts bit 13
+        %279 = OpBitFieldInsert %2 %278 %214 %19 %6 ; inserts bit 14
```

Comment:
The Windows bots are complaining that this string is too big:

```
transformation_add_bit_instruction_synonym_test.cpp(646): error C2026: string too big, trailing characters truncated
```

Can you split the string literal up?  (Or split this into multiple tests.)

Reply:
Sure.

Reply:
Done.